apollo-ios-pagination/Sources/ApolloPagination/GraphQLQueryPager.swift

```diff
@@ -37,22 +60,27 @@ public class GraphQLQueryPager<InitialQuery: GraphQLQuery, PaginatedQuery: Graph
     client: ApolloClientProtocol,
     initialQuery: InitialQuery,
     extractPageInfo: @escaping (PageExtractionData) -> P,
-    nextPageResolver: @escaping (P) -> PaginatedQuery
+    nextPageResolver: ((P) -> PaginatedQuery)?,
+    previousPageResolver: ((P) -> PaginatedQuery)?
```

Comment:
Would it be cleaner to have a single page resolver with a parameter for next/previous?

```swift
enum PaginationDirection {
    case next
    case previous
}

...
pageResolver: ((P, PaginationDirection) -> PaginatedQuery)?
```